tests/test_camx.py

```diff
@@ -0,0 +1,12 @@
+from aqmsp_data.preprocessing import preprocess_camx_met
+import xarray as xr
+
+def test_preprocess_raw_camx():
+    path = "C:/Users/tkrsh/Downloads/camxmet2d.delhi.2023/camxmet2d.delhi.20231018.96hours.nc"
```

Comment:
GitHub runs this workflow online on its server. So it would not know where is `C:/Users/tkrsh/` path? I'd suggest using ubuntu/linux for any kind of development work.